stable/aws-for-fluent-bit/README.md

```diff
@@ -96,15 +116,80 @@ helm delete aws-for-fluent-bit --namespace kube-system
+| `s3.contentType` | A standard MIME type for the S3 object; this will be set as the Content-Type HTTP header. |
+| `s3.sendContentMd5` | Send the Content-MD5 header with PutObject and UploadPart requests, as is required when Object Lock is enabled. | `false`
+| `s3.autoRetryRequests` | Immediately retry failed requests to AWS services once. This option does not affect the normal Fluent Bit retry mechanism with backoff. Instead, it enables an immediate retry with no delay for networking errors, which may help improve throughput when there are transient/random networking issues. | `true`
+| `s3.logKey` | By default, the whole log record will be sent to S3. If you specify a key name with this option, then only the value of that key will be sent to S3. For example, if you are using Docker, you can specify log_key log and only the log message will be sent to S3. |
+| `s3.preserveDataOrdering` | Normally, when an upload request fails, there is a high chance for the last received chunk to be swapped with a later chunk, resulting in data shuffling. This feature prevents this shuffling by using a queue logic for uploads. | `true`
+| `s3.storageClass` | Specify the storage class for S3 objects. If this option is not specified, objects will be stored with the default 'STANDARD' storage class. |`
+| `s3.retryLimit`| Integer value to set the maximum number of retries allowed. Note: this configuration is released since version 1.9.10 and 2.0.1. For previous version, the number of retries is 5 and is not configurable. | 1
+|`s3.externalId`| Specify an external ID for the STS API, can be used with the role_arn parameter if your role requires an external ID.
+|`s3.extraOutputs`| Append extra outputs with value. This section helps you extend current chart implementation with ability to add extra parameters. For example, you can add worker config like`cloudWatchLogs.extraOutputs.Workers=2`|`""`|
+|`opensearch.enabled`| Whether this plugin should be enabled or not, [details](https://docs.fluentbit.io/manual/pipeline/outputs/opensearch) |`true`| ✔
+|`opensearch.match`| The log filter |`"*"`| ✔
+|`opensearch.host`| The url of the Opensearch Search endpoint you want log records sent to. | | ✔
+|`opensearch.awsRegion`| The region in which your Opensearch search is/are in. |`"us-east-1"`|
+|`opensearch.awsAuth`| Enable AWS Sigv4 Authentication for Amazon ElasticSearch Service. |`"off"`|
+|`opensearch.tls`| Enable or disable TLS support | On |
+|`opensearch.port`| TCP Port of the target service. |`443`|
+|`opensearch.path`| OpenSearch accepts new data on HTTP query path "/_bulk". But it is also possible to serve OpenSearch behind a reverse proxy on a subpath. This option defines such path on the fluent-bit side. It simply adds a path prefix in the indexing HTTP POST URI. |`""`|
+|`opensearch.bufferSize`| Specify the buffer size used to read the response from the OpenSearch HTTP service. |`"4k"`|
```

Comment:
I thought I had seen before you set this to a non-default value of `4M` which I liked, because es and opensearch responses are often large and customers do not realize they need to increase this. 

We might not need a value that large but the default 4K is way way too small

Reply:
I missed updating doc